### PR TITLE
Abby

### PR DESCRIPTION
**Pull request by abbymac1**
@bplandry please review

### CODE REVIEW

File: baseHTMLcode [hunk 1]
```diff
@@ -21,6 +21,26 @@
...
-<!--Abby, add a line of code here for practice. Make sure you put a new paragraph in.-->
     </body>
-</html>
+       <head> 
+          <h1>
+           Things I like to bake
+          <h1> 
+          </head>
+      <body>
+        <p>
```
Comment (bplandry) [inline baseHTMLcode:31]: Please remove

File: baseHTMLcode [hunk 2]
```diff
@@ -21,6 +21,26 @@
...
+        <ul>
+            <li>Chocolate cake</li>
+            <li>Chocolate chip cookies</li>
+            <li>Browinies</li>
+         </ul>
+         </p>
```
Comment (bplandry) [inline baseHTMLcode:37]: Please remove

**Review by bplandry — CHANGES_REQUESTED** [review]
Please see my comments. Thanks!

**Comment by abbymac1** [comment]
@bplandry  please review

**Review by bplandry — CHANGES_REQUESTED** [review]
Line 28

**Comment by abbymac1** [comment]
@bplandry please review

**Review by bplandry — APPROVED** [review]

**Pull request merged**